packages/client/src/pages/Map/GoogleMap/legend.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../components/variables';
+
+.legend-wrapper {
+    font-family: $default-font-family;
+    font-size: 1rem;
```

Comment:
There's a `default-font-size` variable as well hehe

Reply:
Haha thanks, I somehow forgot about that